src/components/components/ebay-date-picker-month/index.marko

```diff
@@ -0,0 +1,71 @@
+import {
+    getMonthDisplayName,
+    findVisibleRange,
+    generateDateGrid,
+    getDayLabelFormatter,
+} from './component.js';
+$ const dayLabelFormatter = getDayLabelFormatter(input);
+$ const displayName = getMonthDisplayName(input);
+$ const { showRange, rangeStart, rangeEnd } = findVisibleRange(input);
+$ const dateGrid = generateDateGrid(input);
+
+<div.date-picker__month onKeyDown('onKeyDown')>
+    <h3>${displayName}</h3>
+    <div role='grid' aria-label=displayName>
+        <if(input.weekdayLabels)>
+            <div role='row'>
+                <for|label| of=input.weekdayLabels>
+                    <div.date-picker__day-label role='columnheader'>${label}</div>
+                </for>
+            </div>
+        </if>
+
+        <for|row| of=dateGrid>
+            <div role='row'>
+                <for|date| of=row>
+                    <if(date === undefined)>
+                        <div.date-picker__day--empty/>
+                    </if>
+                    <else>
+                        $ const timestamp = date.getTime();
+                        <div.date-picker__day
```

Comment:
It would be good to move `day` specific logic to a child component?
`ebay-date-picker-day`